thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/task/TaskExecutorFactory.java

```diff
@@ -8,11 +10,12 @@
     Class<?> taskExecutorClass = null;
     switch(taskType) {
       case ANOMALY_DETECTION:
-        taskExecutorClass = AnomalyDetectionTaskRunner.class;
+        taskExecutorClass = DetectionTaskRunner.class;
```

Comment:
Why returning class when you can return the object itself. 
This is TaskExecutorFactory which should create TaskExecutor instances.